c++/AP3.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <cmath>
+
+typedef long long int LL;
+
+using namespace std;
+
+int main() {
+
+        LL t,x,y,sum,i,a,tointeger,newdiff;
```

Comment:
Not a strict rule for competitive coding, but if you are into development, it would be better to follow some style guidelines. Eg. Leave space between comma and new variable name, leave space between operand and operator etc.

Reply:
sure, I'll keep it in mind the next time i make a PR :)